drivers/leds/Kconfig

```diff
@@ -123,6 +123,22 @@ config WS2812_NON_SPI_DRIVER
 		implementations.  Selecting this option builds the new
 		model driver.
 
+config WS2812_RMT_OUTPUT_PIN
+	int "The pin where the ws2812 are connected."
+	depends on WS2812_NON_SPI_DRIVER
+	default 4
+	range 1 35
+	---help---
+		The GPIO pin where the ws2812 are connected.
+
+config WS2812_RMT_CHANNEL
+	int "The RMT's channel used to drive the ws2812"
+	depends on WS2812_NON_SPI_DRIVER
+	default 0
+	range 0 7
+	---help---
+		Which of the RMT's channels that will be used to drive de LEDs.
+
```

Comment:
As already commented by @xiaoxiang781216, `RMT` peripheral is chip-specific and should not be referenced in common code.

Channel configuration should be performed at board level.